src/server/commands.rs

```diff
@@ -1305,4 +1319,25 @@ mod tests {
             assert_eq!(Command::parse(test.input), test.expected);
         }
     }
+
+    #[test]
+    fn parse_mdtm() {
+        struct Test {
+            input: &'static str,
+            expected: Result<Command>,
+        }
+        let tests = [
+            Test {
+                input: "MDTM\r\n",
+                expected: Err(ParseErrorKind::InvalidCommand.into()),
+            },
+            Test {
+                input: "MDTM file.txt\r\n",
+                expected: Ok(Command::MDTM { file: "file.txt".into() }),
+            },
+        ];
```

Comment:
Table driven tests in Rust! :D

Is this idiomatic? I like it!

Reply:
Not sure but it works.